ParisBNF/abb/BNFabb203.xml

```diff
@@ -87,14 +89,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                    <list>
                                       <item xml:id="fol1">
                                          <locus from="1r" to="63r"/>
-                                         Foliation in pencil in the upper right corner of the leaves,
+                                         Foliation made in pencil, in the upper right corner of the leaves,
                                          in Arabic numerals.
                                       </item>
                                    </list>
                                 </foliation>
 
                                 <collation>
-                                   <note>Quire marks (Ethiopic numerals) written regularly, starting with ፪ from f. 9r.</note>
+                                   <note>Quire marks (Ethiopic numerals)
```

Comment:
It would be more precise to include the information on quire marks in `<signatures>` instead of `<note>`, see https://betamasaheft.eu/Guidelines/?q=quire%20marks&id=collationDesc

Reply:
addressed.